gc/base/SweepPoolManagerAddressOrderedListBase.cpp

```diff
@@ -172,7 +173,9 @@ MM_SweepPoolManagerAddressOrderedListBase::connectChunk(MM_EnvironmentBase *env,
 		sweepState->_sweepFreeBytes += leadingFreeEntrySize;
  		sweepState->updateLargestFreeEntry(previousFreeEntrySize, previousPreviousFreeEntry);
 		memoryPool->getLargeObjectAllocateStats()->incrementFreeEntrySizeClassStats(previousFreeEntrySize);
-
+		if ((_extensions->isVLHGC()) && (env->_cycleState->_noCompactionAfterSweep)) {
```

Comment:
I can't quickly understand this part of code, so I don't have a good suggestion for a virtual method name this should go to, but as before bottom line: this code  (and any OMR code that is shared with VLHGC, whenever possible) should be free of any VLHGC specifics, and should go to a VLHGC variant of virtual method.